### PR TITLE
Add "Becoming a Maintainer" link under "Update the recipe" section.

### PR DESCRIPTION
**Pull request by ssurbhi560**
Towards #1331 

PR Checklist:

- [ ] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

**Comment by ssurbhi560** [comment]
@kathatherine @jakirkham @viniciusdc, I was not sure, if I should add the link in the fourth bullet point where it is mentioned that ``Edit the recipe as desired. You may even consider adding yourself as a recipe maintainer.`` or if I should add it at the end of the section. So, I have added it at the end of the section, please let me know what you folks think and if I should add it in the same bullet point.

File: index.html.tmpl [hunk 1]
```diff
@@ -258,6 +258,11 @@
                        </li>
                       </ul>
                     </p>
+                    <p>
+                        If you would like to know how to become a maintainer for a feedstock, please visit <a href="https://conda-forge.org/docs/orga/guidelines.html#becoming-a-maintainer">here</a> to learn more.
```
Comment (kathatherine) [inline index.html.tmpl:262]: I think "go" instead of "visit" would read better.

**Review by kathatherine — CHANGES_REQUESTED** [review]

**Review by kathatherine — APPROVED** [review]
LGTM! @beckermr, any opinions?

**Comment by beckermr** [comment]
LGTM!

**Comment by jakirkham** [comment]
Thanks Surbhi! 🙏

**Pull request merged**